security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/GenerationRedispatchAction.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.action;
+
+import java.util.Objects;
+
+/**
+ * An action increasing or decreasing the active power generation of a generator.
+ *
+ * @author Hadrien Godard <hadrien.godard@artelys.com>
+ */
+public class GenerationRedispatchAction extends AbstractAction {
```

Comment:
Why is it a redispatching? It will be a redispatching if it involves at least two generators, no? Maybe we have to be generic with something like `GeneratorAction`, and a redispatch action will be a list of `GeneratorAction`. What do you think @sylvlecl ?